lib/config/proxy.go

```diff
@@ -114,7 +123,7 @@ func NewConfig() *Config {
 	var cfg Config
 
 	cfg.Proxy.Addr = "0.0.0.0:6000"
-	cfg.Proxy.TCPKeepAlive = true
+	cfg.Proxy.FrontendKeepalive.Cnt = 1
```

Comment:
Set the frontend count to 1? I think the default value is better. That is, 0 means not to update it.